qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/customnavview/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -147,6 +148,13 @@ class MapboxNavigationViewCustomizedActivity : DrawerActivity() {
                 binding.navigationView.api.routeReplayEnabled(isChecked)
             }
         )
+        menuBinding.buttonVoiceInstruction.setOnClickListener {
```

Comment:
Does it make sense to add an example with file and that this file is deleted after by the user? I think it has to be somewhere: here, in the docs, in the examples, etc.

Reply:
We will be adding an example in our public repo. We cam handle the scenario with `File` there.